docs/release-guide.md

```diff
@@ -49,3 +51,13 @@ Pull image from the staging registry.
 ```
 docker pull gcr.io/k8s-staging-descheduler/descheduler:v20200206-0.9.0-94-ge2a23f284
 ```
+
+## Helm Chart
+Helm chart releases are managed by a separate set of git tags that are prefixed with `chart-*`. Example git tag name is `chart-v0.18.0`. Released versions of the
+helm charts are stored in the `gh-pages` branch of this repo. The [chart-releaser-action GitHub Action](https://github.com/helm/chart-releaser-action) is setup to
+build and push the helm charts to the `gh-pages` branch when a `chart-*` git tag is created.
+
+1. Merge all helm chart changes into the appropriate release branch(i.e. release-1.18)
+2. Make sure your repo is clean by git's standards
+3. Create the tag and push it `git checkout release-1.18; CHART_VERSION=chart-v0.18.0; git tag $CHART_VERSION; git push origin $CHART_VERSION`
```

Comment:
Hmmm ... I think the helm chart version is actually 1.0.0. So maybe this should actually be `CHART_VERSION=chart-v1.0.0`?

@stevehipwell what are you thoughts?

Reply:
@seanmalloy I take it from the other commits that you decided to go with the chart aligning to release version.